resdk/query.py

```diff
@@ -239,6 +239,10 @@ def _fetch(self):
             raise NotImplementedError(
                 'Unsupported query_method: {}'.format(self.resource.query_method))
 
+        # Extract data from paginated response
+        if len(items) == 4 and set(items) == set(['next', 'previous', 'count', 'results']):
```

Comment:
This check looks strange. Why not check instead if `items` is a dictionary and it contains a `results` key?